plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -2190,3 +2193,44 @@ def delete_component(self, cid, realm='master'):
         except Exception as e:
             self.module.fail_json(msg='Unable to delete component %s in realm %s: %s'
                                       % (cid, realm, str(e)))
+
+    def get_authz_authorization_scope_by_name(self, name, client_id, realm):
+        url = URL_AUTHZ_AUTHORIZATION_SCOPES.format(url=self.baseurl, client_id=client_id, realm=realm)
+        search_url = "%s/search?name=%s" % (url, name.replace(' ', '%20'))
```

Comment:
```suggestion
        search_url = "%s/search?name=%s" % (url, quote(name))
```
(That function is made for that: https://docs.python.org/3.8/library/urllib.parse.html#urllib.parse.quote)